src/data/data.cc

```diff
@@ -39,15 +122,42 @@ void MetaInfo::Clear() {
   base_margin_.HostVector().clear();
 }
 
+/*
+ * Used format:
+ *
+ * | name        | type     | rows    | columns | value           |
+ * |-------------+----------+---------+---------+-----------------|
+ * | num_row     | kUInt64  | NA      |      NA | ${num_row_}     |
+ * | num_col     | kUInt64  | NA      |      NA | ${num_col_}     |
+ * | num_nonzero | kUInt64  | NA      |      NA | ${num_nonzero_} |
+ * | labels      | kFloat32 | ${size} |       1 | ${labels_}      |
+ * | group_ptr   | kUInt32  | ${size} |       1 | ${group_ptr_}   |
+ * | weights     | kFloat32 | ${size} |       1 | ${weights_}     |
+ * | base_margin | kFloat32 | ${size} |       1 | ${abse_margin}  |
```

Comment:
Please help fixing a typo along the way ;-) . ${abse_margin} -> ${base_margin}.

Reply:
I will.